string detector_ns
string robot_ns
geometry_msgs/PoseStamped pose